src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -304,6 +309,12 @@ simple_data_iterator::compute_properties(
     result.classes = std::move(expected_class_labels);
     int i = 0;
     for (const std::string& label : result.classes) {
+      if (classes_inferred.find(label) == classes_inferred.end() &&
+          is_training_) {
+        std::cout << "Warning: User provided 'classes' includes label '" +
```

Comment:
let me verify whether you can use log_stream to do the warning to stderr.

Reply:
Good!